roles/kernel_cache_drop/tasks/main.yml

```diff
@@ -0,0 +1,32 @@
+---
+- debug:
+    msg: "in cache dropper role"
+
+- name: start kernel cache dropper
+  k8s:
+    definition: "{{ lookup('template', '/opt/kernel_cache_dropper/kernel-cache-drop-daemonset.yaml') | from_yaml }}"
```

Comment:
Moving kernel-cache-drop-daemonset.yaml to the templates dir of the kernel_cache_drop role will avoid to copy it in the Dockerfile and simplify this task

Reply:
OK, that makes sense.